solr/solr-ref-guide/src/rule-based-authorization-plugin.adoc

```diff
@@ -64,6 +66,34 @@ There are several things defined in this example:
 <5> The 'admin' role has been defined, and it has permission to edit security settings.
 <6> The 'solr' user has been defined to the 'admin' role.
 
+=== Example for ExternalRoleRuleBasedAuthorizationPlugin
+This example `security.json` shows how an imagined `JWTAuthPlugin`, which pulls user and user roles from JWT claims, can work with the `ExternalRoleRuleBasedAuthorizationPlugin` plugin:
```

Comment:
s/imagined//, this exists, right?

Reply:
I brought back some of the refguide changes from the original patch. Please review again.